### PR TITLE
2.068.0fix check for nan

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

**Comment by 9il** [comment]
Can you move change to `erfcx`?

``` D
        return 
            x < -26.7 ? 
                double.infinity :
                x < -6.1 ? 
                    2*exp(x*x) :
                    x.isNaN ?
                        x :
                        2*exp(x*x) - erfcx_y100(400/(4-x));
```

**Comment by John-Colvin** [comment]
How's that?

File: source/libcerf/erfcx_.d [hunk 1]
```diff
@@ -65,13 +65,10 @@ double erfcx(in double x) @safe @nogc nothrow
...
-    else
-        return 
-            x < -26.7 ? 
-                double.infinity :
-                x < -6.1 ? 
-                    2*exp(x*x) :
-                    2*exp(x*x) - erfcx_y100(400/(4-x));
+    else if(x < -26.7) return double.infinity;
+    else if(x < -6.1)  return 2*exp(x*x);
+    else if(x.isNaN)   return double.nan;
```
Comment (9il) [inline source/libcerf/erfcx_.d]: `else if(x.isNaN) return x`, this code is faster and saves payload.

Comment (John-Colvin) [inline source/libcerf/erfcx_.d]: Understood. Also, it preserved the specific nan as opposed to the generic `.nan`

**Pull request merged**